src/ir/types/schema_test.cc

```diff
@@ -0,0 +1,72 @@
+#include "src/ir/types/schema.h"
+#include "src/ir/types/proto/schema.h"
+#include "src/ir/types/type.h"
+#include "src/ir/types/entity_type.h"
+
+
+#include "google/protobuf/text_format.h"
+#include "src/common/testing/gtest.h"
+#include "absl/strings/str_cat.h"
+
+
+namespace raksha::ir::types {
+
+
+
+    class ToStringTest: public testing::TestWithParam<std::tuple<std::string, std::string>> {
```

Comment:
formatting and indentation is off in this file.

Reply:
This is still off? Can you use `clang-format` to fix it? You can add `clang-format` as a plugin in VS code.

Reply:
Oops, sorry! I thought I just needed to install it and configure the settings, but I needed to set the 'onSave' config as well. I think it's resolved now.